bookmarks/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from bookmarks.views import Create_Bookmarks
+from bookmarks.views import Remove_bookmarks
+
+
+app_name = 'bookmarks'
+
+urlpatterns = [
+    path("", Create_Bookmarks, name="Create_Bookmarks"),
+    path("Remove_bookmarks/<int:pk>/", Remove_bookmarks, name="Remove_bookmarks"),
+]
```

Comment:
Do you really want to have capital letters in your URL?
being that these are under bookmarks/... maybe it can just be create and remove instead